### PR TITLE
Use unicode spec directly instead of emojione json

### PR DESCRIPTION
**Pull request by suprMax**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,10 +43,12 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
     "express": "^4.17.1",
+    "got": "^9.6.0",
+    "lodash.difference": "^4.5.0",
     "lodash.uniq": "^4.5.0",
+    "mkdirp": "^0.5.1",
```
Comment (atomiomi) [inline package.json]: u can pass `recursive: true` to `fs.mkdir` in nodejs 10

File: preview.js [hunk 2]
```diff
@@ -84,30 +83,28 @@ const renderCollection = () => {
...
-    res.send(html);
-  };
-
-  request(URL, handleSpecReady);
+const handleSpec = async(req, res) => {
+  const spec = await getUnicodeSpec(getVersion());
+
+  const specUnicode = spec.map(({ unicode }) => unicode);
+  const specUnicodeImages = specUnicode.map((unicode) => renderEmoji(unicode, renderOptions));
+  const specCodePointsImages = spec.map(({ codePoint }) => (
```
Comment (atomiomi) [inline preview.js]: generate these 3 arrays in 1 cycle?

File: tasks/generate.js [hunk 3]
```diff
@@ -72,55 +42,54 @@ const isSuggestable = (key) => {
...
 };
 
-const getCollection = () => {
-  const keys = getKeys();
+const getEmojiData = (spec, assets) => spec.reduce((acc, item) => {
+  const { codePoint: hex, unicode, qualified } = item;
+  const key = hexToId(hex);
+
+  if (!assets[key]) {
+    console.error(`Coundn't find ${key} in the assets data`);
```
Comment (atomiomi) [inline tasks/generate.js]: warn?

File: tasks/utils/assets.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const pkg = require('../../package');
+
+const version = pkg.devDependencies['emojione-assets'].replace(/\^/g, '');
+
+const utils = {
+  getVersion() {
```
Comment (atomiomi) [inline tasks/utils/assets.js]: `exports.getVersion = ...`?

File: tasks/utils/data.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const got = require('got');
+
+const { logError } = require('./log');
+const { parseUnicodeSpec } = require('./unicode');
+
+const OUTPUT = path.resolve(`${__dirname}/../../build`);
```
Comment (atomiomi) [inline tasks/utils/data.js]: no need in interpolation, `resolve(__dirname, '../../build')` should do it

File: preview.js (hunk 2 above)
Comment (suprMax) [reply preview.js]: performance here is irrelevant

File: tasks/generate.js (hunk 3 above)
Comment (suprMax) [reply tasks/generate.js]: they all look the same atm afaik

File: tasks/utils/assets.js (hunk 4 above)
Comment (suprMax) [reply tasks/utils/assets.js]: we always reassign exports

File: package.json (hunk 1 above)
Comment (suprMax) [reply package.json]: won't work, needs `-p`

**Pull request merged**